pages/common/pdftex.md

```diff
@@ -0,0 +1,16 @@
+# pdftex
+
+> Compile a PDF document from TeX source files.
+> More information: <https://linux.die.net/man/1/pdftex>.
```

Comment:
we try to avoid `linux.die`. Can we use the Arch Linux pages for this commands? @bl-ue 
```suggestion
> More information: <https://man.archlinux.org/man/pdftex.1>.
```

Reply:
https://github.com/tldr-pages/tldr/pull/5294#issuecomment-782873074

Reply:
Oh 😅, yeah I think we should use the websites.